wdn/templates_4.1/scripts/main-wdn-plugins.js

```diff
@@ -1,8 +1,8 @@
 define([
 	'analytics',
+	'socialmediashare',
 	'navigation',
 	'search',
-	'socialmediashare',
```

Comment:
Sure, but you _should not_ depend on that. Dependencies should be explicitly declared and initialization should be entirely independent of the status of another module.